client/src/cmdhficlass.c

```diff
@@ -4472,18 +4472,18 @@ static int CmdHFiClassSAM(const char *Cmd) {
 
 static command_t CommandTable[] = {
     {"help",        CmdHelp,                    AlwaysAvailable, "This help"},
-    {"list",        CmdHFiClassList,            AlwaysAvailable, "List iclass history"},
+    {"list",        CmdHFiClassList,            IfPm3Iclass,     "List iclass history"},
```

Comment:
this can be used offline.

Reply:
Can you provide more insights on this?
On my end this is what happens when using this offline:

<img width="772" alt="image" src="https://github.com/RfidResearchGroup/proxmark3/assets/8178359/eb176bd4-2e5f-444f-abb3-429c077e9fae">

Reply:
Like the output say,   you haven't loaded a trace list so there is nothing to show.

Reply:
Reverted the change for this one, will upload shortly once I finish the checks on the encode function